vispy/gloo/glir.py

```diff
@@ -497,7 +497,14 @@ def _gl_initialize(self):
             self.capabilities['gl_version'] = gl.glGetParameter(gl.GL_VERSION)
             self.capabilities['max_texture_size'] = \
                 gl.glGetParameter(gl.GL_MAX_TEXTURE_SIZE)
-            this_version = self.capabilities['gl_version'].split(' ')[0]
+            this_version = self.capabilities['gl_version'].split(' ')
+            if this_version[0] == "OpenGL":
+                # For OpenGL ES, the version string has the format:
+                # "OpenGL ES <version number> <vendor-specific information>"
+                this_version = this_version[2]
```

Comment:
Is this difference documented in the official standards? Do you have a link?

Reply:
Yes, the ES version format string is specified in the description for GL_VERSION in the ES2 reference:

https://www.khronos.org/registry/OpenGL-Refpages/es2.0/xhtml/glGetString.xml 

OpenGL<space>ES<space><version number><space><vendor-specific information>